packages/cardano/src/wallet/lib/get-inputs-value.ts

```diff
@@ -1,61 +1,34 @@
 import { Cardano, ChainHistoryProvider } from '@cardano-sdk/core';
 import { createWalletUtil, ObservableWallet } from '@cardano-sdk/wallet';
-import flattenDeep from 'lodash/flattenDeep';
-
-const TX_PAGINATION_LIMIT = 25;
 
 export type TxInput = { value?: Cardano.Value; address?: Cardano.HydratedTxIn['address'] } & Pick<
   Cardano.HydratedTxIn,
   'index' | 'txId'
 >;
 
-const fetchTransactionByHashes = async (chainProviderInstance: ChainHistoryProvider, ids: Cardano.TransactionId[]) => {
-  if (ids.length <= TX_PAGINATION_LIMIT) {
-    return chainProviderInstance.transactionsByHashes({ ids });
-  }
-
-  const paginatedPromises = [];
-  for (let i = 0; i < ids.length; i += TX_PAGINATION_LIMIT) {
-    paginatedPromises.push(chainProviderInstance.transactionsByHashes({ ids: ids.slice(i, i + TX_PAGINATION_LIMIT) }));
-  }
-  return flattenDeep(await Promise.all(paginatedPromises));
-};
-
 export const getTxInputsValueAndAddress = async (
   inputs: Cardano.HydratedTxIn[] | Cardano.TxIn[],
   chainProviderInstance: ChainHistoryProvider,
   wallet: ObservableWallet
```

Comment:
It would be a little nicer if this function was just `resolveInputs`, but it's also fine as-is too because creating wallet util should be cheap anyway
```suggestion
  resolver: InputResolver
```